<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LessonWindow</class>
 <widget class="QDialog" name="LessonWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выбор теста</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="welcomeLabel">
     <property name="styleSheet">
      <string notr="true">font-size: 16pt; font-weight: bold; color: #2c3e50; padding: 10px;</string>
     </property>
     <property name="text">
      <string>Добро пожаловать, [Имя ученика]</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="averageScoreLabel">
     <property name="styleSheet">
      <string notr="true">font-size: 12pt; color: #7f8c8d; margin-bottom: 10px;</string>
     </property>
     <property name="text">
      <string>Средний балл: 0.0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="searchLayout">
     <item>
      <widget class="QLineEdit" name="searchField">
       <property name="styleSheet">
        <string notr="true">padding: 8px; border: 1px solid #bdc3c7; border-radius: 4px;</string>
       </property>
       <property name="placeholderText">
        <string>Поиск по названию или предмету...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    padding: 8px 15px;
    background-color: #3498db;
    color: white;
    border: none;
    border-radius: 4px;
}
QPushButton:hover {
    background-color: #2980b9;
}</string>
       </property>
       <property name="text">
        <string>Поиск</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refreshButton">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    padding: 8px 15px;
    background-color: #27ae60;
    color: white;
    border: none;
    border-radius: 4px;
}
QPushButton:hover {
    background-color: #229954;
}</string>
       </property>
       <property name="text">
        <string>Обновить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="testsList">
     <property name="styleSheet">
      <string notr="true">QListWidget {
    background-color: white;
    border: 1px solid #bdc3c7;
    border-radius: 4px;
    color: black;
    padding: 5px;
}
QListWidget::item {
    padding: 12px;
    border-bottom: 1px solid #ecf0f1;
    background-color: white;
}
QListWidget::item:selected {
    background-color: #3498db;
    color: white;
    border-radius: 3px;
}</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QLabel" name="infoLabel">
       <property name="styleSheet">
        <string notr="true">color: #7f8c8d; font-style: italic;</string>
       </property>
       <property name="text">
        <string>Доступно тестов: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="startTestButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    padding: 10px 20px;
    font-weight: bold;
    background-color: #e74c3c;
    color: white;
    border: none;
    border-radius: 4px;
    font-size: 12pt;
}
QPushButton:hover {
    background-color: #c0392b;
}
QPushButton:pressed {
    background-color: #a93226;
}
QPushButton:disabled {
    background-color: #95a5a6;
}</string>
       </property>
       <property name="text">
        <string>Начать тест</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    padding: 10px 20px;
    background-color: #95a5a6;
    color: white;
    border: none;
    border-radius: 4px;
}
QPushButton:hover {
    background-color: #7f8c8d;
}</string>
       </property>
       <property name="text">
        <string>Выйти</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>LessonWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
